graphql_schema.json

```diff
@@ -1160,6 +1270,37 @@
                   "ofType": null
                 }
               },
+              "isDeprecated": true,
+              "deprecationReason": "use unlockAccount instead"
+            },
+            {
+              "name": "lockAccount",
+              "description": "Lock an unlocked account to prevent transaction being sent from it",
```

Comment:
qq -- can you lock an already locked account?

Reply:
Uh problably